docs/source/notebooks/SMC-ABC_Lotka-Volterra_example.ipynb

```diff
@@ -80,7 +80,7 @@
    "outputs": [],
```

Comment:
<p>Maybe add a last cell with <code>watermark</code> output, like in resources repo? Finally, can you run <code>black_nbconvert</code> please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3940/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>